### PR TITLE
TLT-3843 Dockerize CAAT

### PR DESCRIPTION
**Pull request by Chris-Thornton-Harvard**
Converts the project to be able to run in Docker

Currently deployed to QA

### CODE REVIEW

File: canvas_account_admin_tools/requirements/base.txt [hunk 1]
```diff
@@ -37,3 +39,6 @@ redis==3.3.8
 requests==2.22.0
 simplejson==3.10.0
 git+ssh://git@github.com/Harvard-University-iCommons/django-ssm-parameter-store.git@v0.3#egg=django-ssm-parameter-store==0.3
+
+splunk_handler==2.2.0
+python-json-logger==0.1.11
```
Comment (elliottyates) [inline canvas_account_admin_tools/requirements/base.txt:44]: Any reason we're not using a newer versions?
* https://github.com/zach-taylor/splunk_handler/releases
* https://github.com/madzak/python-json-logger/blob/master/CHANGELOG.md

File: canvas_account_admin_tools/settings/base.py [hunk 2]
```diff
@@ -235,52 +237,75 @@
...
             'level': _DEFAULT_LOG_LEVEL,
-            'formatter': 'verbose',
-            'filename': os.path.join(_LOG_ROOT, 'django-canvas_account_admin_tools.log'),
+            'filters': ['context'],
+        },
+        'gunicorn': {
+            'class': 'splunk_handler.SplunkHandler',
+            'formatter': 'json',
+            'sourcetype': 'json',
+            'source': 'gunicorn-canvas_manage_course',
```
Comment (elliottyates) [inline canvas_account_admin_tools/settings/base.py:295]: typo?

**Review by elliottyates — APPROVED** [review]
👍 lgtm!

**Comment by Chris-Thornton-Harvard** [comment]
I am going to add in the build info as an HTML comment before merging this btw

**Pull request merged**